acceptance/lib/puppet/acceptance/common_utils.rb

```diff
@@ -1,24 +1,10 @@
-require 'puppet/acceptance/puppet_helpers'
-extend Puppet::Acceptance::PuppetHelpers
-
 module Puppet
   module Acceptance
-    module CAUtils
```

Comment:
So if we're copying in artefacts from other projects, it surprises me a little that facter is the project of record for the namespace `Puppet::Acceptance`.  Is this Puppet in the sense of 'Puppet the company', or was it just a convenient place to grab it from as it should be the same across puppet and facter repos?

Actually.  This is fairly icky too:  https://github.com/puppetlabs/puppet/blob/master/acceptance/lib/puppet/acceptance/common_utils.rb in puppet's tree that one file declares `Puppet::Acceptance::{CronUtils,CAUtils, CommandUtils}`, which though it explains 'common' in `common_utils.rb` to mean  'random grab bag', is still fairly poor mapping from namespace to file system space.

If we're confident that this PR is self-consistent I'm happy to merge and then continue the discussion about shared test library hygene and lifecycle in some other avene.